File: contest/joined/codeforces-1846/A.cpp
// Author : ysh
// 2023/07/07 Fri 22:39:12
#include<bits/stdc++.h>
using namespace std;
int main() {
    ios::sync_with_stdio(false);
    cin.tie(0);

    int n;cin>>n;
    while(n--) {
        int n;cin>>n;
        int ans = 0;
        while(n--) {
            int a,b;cin>>a>>b;
            if(a - b > 0) ans++;
        }
        cout<<ans<<"\n";
    }
}

File: contest/joined/codeforces-1846/A.py
# Author : ysh
# 2023/07/07 Fri 22:36:57
for i in range(int(input())):
    n = []
    ans = 0
    for j in range(int(input())):
        n.append([int(i) for i in input().split()])
        if n[j][0] - n[j][1] > 0:
            ans = ans + 1
    print(ans)
quit()